pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -208,6 +208,12 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 				return r.setHibernatingCondition(cd, hivev1.SyncSetsNotAppliedReason, "Cluster SyncSets have not been applied", corev1.ConditionFalse, cdLog)
 			}
 		}
+
+		// Hibernate fake cluster if hibernate after is not set
+		if controllerutils.IsFakeCluster(cd) && cd.Spec.HibernateAfter == nil && hibernatingCondition.Status != corev1.ConditionTrue {
+			r.setHibernatingCondition(cd, hivev1.HibernatingHibernationReason, "Fake cluster is stopped",
```

Comment:
This chunk is [not being covered by UT](https://app.codecov.io/gh/openshift/hive/compare/1477/tree/pkg/controller/hibernation/hibernation_controller.go#D1L214) -- see below.